### DISCUSSION TITLE
Is there a way to connect to a remote python repl?

### DISCUSSION THREAD

**Discussion by qnkhuat** [Olical/conjure#629] [Q&A]
like the tile, I wonder if I can vim with conjure locally but evaluate it on a remote python repl?

**Comment by Olical** [comment]
You can't connect to one that's running because we run Python as a sub-process under Neovim and talk to it over stdio. If there is a Python implementation of a networked REPL, then maybe we can adopt that as an option soon which could work over a network like the Clojure nREPL solution does for example.

If you want to eval on a remote machine, you could try overriding the command that Conjure launches for the Python sub-process and replace it with an `ssh` command that starts a Python REPL on a remote host. I haven't tried this and I'm not sure what args you'd need to make it work... but it might work? Worth a shot!
